src/main/java/ro/msg/learning/shop/dto/LocationDTO.java

```diff
@@ -0,0 +1,33 @@
+package ro.msg.learning.shop.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import ro.msg.learning.shop.model.Coordinates;
+
+import javax.swing.*;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class LocationDTO {
+    private Coordinates latLng;
+    private String adminArea4;
+    private String adminArea5Type; //CITY
```

Comment:
I guess here you named the fields exactly in the same way as the route matrix API returns it's results. You should actually just declare the fields you are interested in and ignore the rest (use `@JsonIgnoreProperties(ignoreUnknown = true)` on the class). Also, you should give the fields your own names and customize the way in which the fields are deserialized using the `@JsonProperty("fieldName")` annotation on each field.